test/controllers/landing-page.controller.test.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// For running our service
+const { init } = require('../../app/server.js')
+
+describe('Landing Page controller', () => {
+  let server
+
+  // Create server before each test
+  beforeEach(async () => {
+    server = await init()
```

Comment:
Looking at the config for the controller tests in other files, it appears we need to add Sinon to stub the `server.logger ` and `server.app.airbrake`. By doing this we are silencing any notification that might get sent to Errbit.